src/util.ts

```diff
@@ -5,7 +5,7 @@ interface LocalStored {
   "resync-volume": number
   "resync-muted": boolean
   "resync-last-room": string
-  "secret": string
+  secret: string
```

Comment:
wait, why is this diefferent than the other local storage values?

Reply:
the quotes are only required when the property name includes special characters like `-`.
but yea i'll change it to resync-secret in the next commit.